test/models/opensearch_test.rb

```diff
@@ -279,6 +279,8 @@ class OpensearchTest < ActiveSupport::TestCase
     os.instance_variable_set(:@params,
                              { geodistance: { latitude: '42.361145', longitude: '-71.057083', distance: '50mi' } })
 
+    refute(os.matches.to_json.include?('{"multi_match":{"query":"rail stations","fields":'))
```

Comment:
The intent of this test is to confirm that the query builder isn't hallucinating a keyword search? I think I can understand that motivation, but wonder if there's a smaller string match to look for - like maybe refute on just `multi_match` or something. The way this reads right now, I think hallucinating a search for popcorn would pass the test - as long as it doesn't hallucinate rail stations?

Reply:
I narrowed it down and am just squashing/merging without a review as it was so minor. Thanks again.